### PR TITLE
Migration docs

### PR DESCRIPTION
**Pull request by maimaisie**
###### Description

Adding docs for migration from old to new setup.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -739,6 +742,43 @@ RUN gem install fluent-plugin-aws-elasticsearch-service
...
+## Delete the resources associated with Fluentd
+
+Find the fluentd daemonset:
+```
+kubectl get daemonsets
+```
+
+Delete the Fluentd daemonset, configmap and secret:
+```
+kubectl delete daemonset fluentd-sumologic
```
Comment (frankreno) [inline deploy/README.md]: What about the RBAC configuration?

File: deploy/README.md [hunk 2]
```diff
@@ -739,6 +742,43 @@ RUN gem install fluent-plugin-aws-elasticsearch-service
...
+kubectl delete configmap fluentd-sumologic-config
+kubectl delete secret sumologic
+```
+
+## Delete the resources associated with Heapster
+
+Delete the service, deployment and configmap for Heapster:
+```
+kubectl delete service sumo-graphite
```
Comment (frankreno) [inline deploy/README.md]: Same here for RBAC configuration.

File: deploy/README.md [hunk 3]
```diff
@@ -739,6 +742,43 @@ RUN gem install fluent-plugin-aws-elasticsearch-service
...
+kubectl delete deployment sumo-graphite
```
Comment (frankreno) [inline deploy/README.md]: The problem is you also need to decide what to do with heapster.  If you look at our old steps, you modify heapster to output to the sumo service.  If you delete the deployment without cleaning that up, heapster breaks and starts spewing errors.  We should add a step that shows them what they need to remove from their Heapster command

File: deploy/README.md (hunk 1 above)
Comment (maimaisie) [reply deploy/README.md]: good catch, will add

File: deploy/README.md (hunk 2 above)
Comment (maimaisie) [reply deploy/README.md]: I did not see any RBAC setting from the old repo readme for Heapster https://github.com/SumoLogic/fluentd-kubernetes-sumologic#step-4-set-up-heapster-for-metric-collection
Is there something I'm missing

Comment (frankreno) [reply deploy/README.md]: ahh you know what you are right, we are good there, my bad

File: deploy/README.md [hunk 4]
```diff
@@ -739,6 +742,52 @@ RUN gem install fluent-plugin-aws-elasticsearch-service
 USER fluent
 ```
 
+# Migration Steps
+
+Our [old collection mechanism](https://github.com/SumoLogic/fluentd-kubernetes-sumologic) includes Fluentd for logs and Heapster for metrics.
```
[the remark this answers is not in the document]
Comment (maimaisie) [reply deploy/README.md]: sure

File: deploy/README.md [hunk 5]
```diff
@@ -739,6 +742,52 @@ RUN gem install fluent-plugin-aws-elasticsearch-service
...
+* If you plan to deploy the new collection mechanism to the same namespace where the old mechanism was deployed to, you will need to delete the old set up first.
```
[the remark this answers is not in the document]
Comment (drduke1) [reply deploy/README.md]: @maimaisie Yes or no on this?

[the remark this answers is not in the document]
Comment (maimaisie) [reply deploy/README.md]: oh sorry I meant to reply yesterday, yes we can change to that

File: deploy/README.md (hunk 3 above)
Comment (maimaisie) [reply deploy/README.md]: I assume this would just be removing the sink option?

**Pull request merged**